Source/UI/CountrySelection/CountrySelectionViewController.swift

```diff
@@ -34,7 +34,9 @@ public final class CountrySelectionViewController: UIViewController,
     /// Called after the controller's view is loaded into memory.
     override public func viewDidLoad() {
         super.viewDidLoad()
-        customizeNavigationBarAppearance(color: .white, titleColor: .black)
+        if #available(iOS 13.0, *) {
+            view?.overrideUserInterfaceStyle = .light
+        }
```

Comment:
Any reason we don't just disable for ["Entire App via Info.plist"](https://stackoverflow.com/a/56546554)?

Given that would require no code level changes I would think it to be more favourable. Also pretty straight forward.

Reply:
Yes, Every project(SPM, PODS, Carthage) has it's own info.plist so this line is a common class like `AppDeleagte` will have impact on all projects.